### ISSUE TITLE
[Docker compose] Error with connection to s3 provider

### ISSUE THREAD

**Issue by NikolaBoyko** [kerberos-io/vault#11]
Hi, i have a problem with connection to s3 (mino server) and storj provider.
S3:
![Screenshot from 2023-05-29 01-28-06](https://github.com/kerberos-io/vault/assets/19174107/57132bdc-a8cf-4ac1-8f9a-18620bb6db71)
Storj:
![Screenshot from 2023-05-29 01-35-04](https://github.com/kerberos-io/vault/assets/19174107/d8e7ed97-3843-475d-b727-72d3edab2485)

The Vault binary does not have any debug features because I can't find any errors in the log?

**Comment by cedricve** [comment]
How does your configuration looks like?

**Comment by NikolaBoyko** [comment]
```
version: "3.9"

services:
  kerberos-vault:
    image: 'kerberos/vault:1.0.871485739'
    restart: always
    networks:
      - vault-stack
      - web
    volumes:
      - "./vault:/vault"
      - "./entrypoint.sh:/home/vault/entrypoint.sh:ro"
      - "./wait-for.sh:/home/vault/wait-for.sh:ro"
    environment:
      - GIN_MODE=release
      - KERBEROS_LOGIN_USERNAME=user
      - KERBEROS_LOGIN_PASSWORD=pass
      - MONGODB_HOST=kerberos-mongo
      - MONGODB_DATABASE_CREDENTIALS=admin
      - MONGODB_USERNAME=kerberos
      - MONGODB_PASSWORD=pass
      - MONGODB_DATABASE_STORAGE=KerberosVault
      - CONTINUOUS_FORWARDING=false
#    command: './entrypoint.sh'
    command: sh -c './wait-for.sh kerberos-mongo:27017 -- ./entrypoint.sh'
    depends_on:
      - kerberos-mongo

  kerberos-mongo:
    image: 'mongo:4.4.16'
    restart: always
    networks:
      - vault-stack
    volumes:
      - ./mongodb:/data/db
    environment:
      - MONGO_INITDB_ROOT_USERNAME=kerberos
      - MONGO_INITDB_ROOT_PASSWORD=pass
    command: --wiredTigerCacheSizeGB 1

networks:
  vault-stack: {}
  web:
    external: true
```

**Comment by cedricve** [comment]
I mean the provider settings, from the Kerberos Vault UI.

**Comment by NikolaBoyko** [comment]
![Screenshot from 2023-05-29 02-06-15](https://github.com/kerberos-io/vault/assets/19174107/c91a9dc0-0609-4692-a0dc-6eaec00517c0)

**Comment by cedricve** [comment]
<img width="803" alt="Screenshot 2023-05-29 at 22 35 40" src="https://github.com/kerberos-io/vault/assets/1546779/1d1d1149-7c92-402a-86c6-3886931651c4">

Not sure what you have done differently?

**Comment by cedricve** [comment]
As referenced above (https://github.com/kerberos-io/agent/issues/101) this might be caused due to `docker compose`, investigating what is going on.

**Comment by NikolaBoyko** [comment]
Hi! Problem was resolved in container version kerberos/vault:1.0.882646955. I think fix is identical to [kerberos-io/agent#101](https://github.com/kerberos-io/agent/issues/101)
